beacon_node/beacon_chain/src/data_availability_checker.rs

```
@@ -445,8 +445,6 @@ impl<T: BeaconChainTypes> DataAvailabilityChecker<T> {
             .flatten()
             .map(CustodyDataColumn::into_inner)
             .collect::<Vec<_>>();
-        let all_data_columns =
-            RuntimeVariableList::from_vec(all_data_columns, T::EthSpec::number_of_columns());
```

This silently truncate all data columns to `number_of_columns`

---

We should probably remove these sorts of silent truncation functions, there are similar ones on Variable List

---

Fixed in a separate PR